modules/platform/nuxeo-easyshare-core/src/main/java/org/nuxeo/easyshare/EasyShareEncryptionUtil.java

```diff
@@ -0,0 +1,180 @@
+package org.nuxeo.easyshare;
+
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.AlgorithmParameterSpec;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.spec.PBEParameterSpec;
+
+import org.apache.commons.codec.binary.Base64;
+
+public class EasyShareEncryptionUtil {
```

Comment:
Could you add:
```suggestion
/**
 * @since 11.5
 */
public class EasyShareEncryptionUtil {
```

Reply:
It seems that the whole file contains tabs, could you replace them by spaces?

Reply:
Done for "since 11.5" and re-format using nuxeo formater